src/modules/localStorage.js

```diff
@@ -0,0 +1,18 @@
+export const STORAGE_KEY = 'todos';
+
+export function readTask() {
+  const todos = JSON.parse(localStorage.getItem(STORAGE_KEY)) || [];
+  return todos;
+}
+
+export function updateTask(index, updatedTask) {
+  const tasks = readTask();
+  tasks[index] = updatedTask;
+  localStorage.setItem(STORAGE_KEY, JSON.stringify(tasks));
+}
+
+export function deleteTask(index) {
+  const todos = readTask();
+  todos.splice(index, 1);
+  localStorage.setItem(STORAGE_KEY, JSON.stringify(todos));
+}
```

Comment:
- [ ] You have done a great job 👍of implementing the delete functionality. However,🙆‍♀️ when you delete a todo, the indexes are not reordered. 🙏Please add code to reorder your indexes as instructed in the project requirements. This will ensure that your local storage has proper index order. Please refer to the project requirements below:
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..

Reply:
Well Resolved ✔️ 

[OPTIONAL] The value of the first task `index` should start at 1. Currently, the task index starts at 0.